src/duqtools/data/variables.yaml

```diff
@@ -11,279 +13,288 @@
   type: IDS-variable
 - name: t_e
   ids: core_profiles
   path: profiles_1d/*/electrons/temperature
-  dims: [time, x]
+  dims: [time, $rho_tor_norm]
   type: IDS-variable
 - name: p_e
   ids: core_profiles
   path: profiles_1d/*/electrons/pressure
-  dims: [time, x]
+  dims: [time, $rho_tor_norm]
   type: IDS-variable
 - name: n_i_tot
   ids: core_profiles
   path: profiles_1d/*/n_i_thermal_total
-  dims: [time, x]
+  dims: [time, $rho_tor_norm]
   type: IDS-variable
 - name: n_i
   ids: core_profiles
   path: profiles_1d/*/ion/*/density_thermal
-  dims: [time, ion, x]
+  dims: [time, ion, $rho_tor_norm]
   type: IDS-variable
 - name: n_i_tot
   ids: core_profiles
   path: profiles_1d/*/ion/*/density
-  dims: [time, ion, x]
+  dims: [time, ion, $rho_tor_norm]
   type: IDS-variable
 - name: p_i
   ids: core_profiles
   path: profiles_1d/*/ion/*/pressure
-  dims: [time, ion, x]
+  dims: [time, ion, $rho_tor_norm]
   type: IDS-variable
 - name: n_n
   ids: core_profiles
   path: profiles_1d/*/neutral/*/density
-  dims: [time, neutral, x]
+  dims: [time, neutral, $rho_tor_norm]
   type: IDS-variable
 - name: j_tot
   ids: core_profiles
   path: profiles_1d/*/j_total
-  dims: [time, x]
+  dims: [time, $rho_tor_norm]
   type: IDS-variable
 - name: j_ohm
   ids: core_profiles
   path: profiles_1d/*/j_ohmic
-  dims: [time, x]
+  dims: [time, $rho_tor_norm]
   type: IDS-variable
 - name: j_ni
   ids: core_profiles
   path: profiles_1d/*/j_non_inductive
-  dims: [time, x]
+  dims: [time, $rho_tor_norm]
   type: IDS-variable
 - name: q
   ids: core_profiles
   path: profiles_1d/*/q
-  dims: [time, x]
+  dims: [time, $rho_tor_norm]
   type: IDS-variable
 - name: smag
   ids: core_profiles
   path: profiles_1d/*/magnetic_shear
-  dims: [time, x]
+  dims: [time, $rho_tor_norm]
   type: IDS-variable
 - name: e_r
   ids: core_profiles
   path: profiles_1d/*/e_field/radial
-  dims: [time, x]
+  dims: [time, $rho_tor_norm]
   type: IDS-variable
 - name: e_par
   ids: core_profiles
   path: profiles_1d/*/e_field/parallel
-  dims: [time, x]
+  dims: [time, $rho_tor_norm]
   type: IDS-variable
 - name: omega_tor
   ids: core_profiles
   path: profiles_1d/*/rotation_frequency_tor_sonic
-  dims: [time, x]
+  dims: [time, $rho_tor_norm]
   type: IDS-variable
 - name: collisionality
   ids: core_profiles
   path: profiles_1d/*/electrons/collisionality_norm
-  dims: [time, x]
+  dims: [time, $rho_tor_norm]
   type: IDS-variable
+
+### Core sources
+
 - name: energy_e_src
   ids: core_sources
   path: source/0/profiles_1d/*/electrons/energy
-  dims: [time_src, x]
+  dims: [time_src, $rho_tor_norm_src]
   type: IDS-variable
 - name: particles_e_src
   ids: core_sources
   path: source/0/profiles_1d/*/electrons/particles
-  dims: [time_src, x]
+  dims: [time_src, $rho_tor_norm_src]
   type: IDS-variable
 - name: energy_i_src
   ids: core_sources
   path: source/0/profiles_1d/*/ion/*/energy
-  dims: [time_src, ion_src, x]
+  dims: [time_src, ion_src, $rho_tor_norm]
```

Comment:
is this correct, shouldnt it be `rho_tor_norm_src`?

Reply:
Yes, good catch 👍